wpeverest/everest-forms
# EVF 197 Enhancement - Four column Form

### All Submissions:



### Changes proposed in this Pull Request:
Previously,There was two column Form this PR enhance 4 Column Form

### How to test the changes in this Pull Request:

1. Create a form with 3 and 4 Column.
2. Verify the changes in Builder and front end from your side.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Four column Form
